flatfs_test.go

```diff
@@ -429,6 +429,15 @@ func testDiskUsage(dirFunc mkShardFunc, t *testing.T) {
 	}
 	t.Log("duPostDelete:", duDelete)
 
+	// Make sure DiskUsageNotes files exists and the correct value
+	d, err := ioutil.ReadFile(filepath.Join(temp, flatfs.DiskUsageNotes))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if string(d) != "initial-exact\n" {
```

Comment:
Use constant instead of literal?

Reply:
It's a test, so I want to make sure it has the right value.

Reply:
But the right value is and will always be the one defined by the constant?

Reply:
Yes, unless its accidentally modified in which case this test will fail to alert you of that fact.  If it was intentional, then the test case (and the README) should also be modified.